firefox-ios/firefox-ios-tests/Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -107,8 +107,12 @@ class JumpBackInTests: BaseTestCase {
         app.cells["JumpBackInCell"].staticTexts["Wikipedia"].firstMatch.tap()
 
         // The view is switched to the twitter tab
-        let url = app.textFields[AccessibilityIdentifiers.Browser.AddressToolbar.searchTextField].value as! String
-        XCTAssertEqual(url, "wikipedia.org")
+        if let url = app.textFields[AccessibilityIdentifiers.Browser.AddressToolbar.searchTextField].value as? String {
+            XCTAssertEqual(url, "wikipedia.org")
+        } else {
+            XCTFail("Failed to retrieve the URL string from the address toolbar")
+            return
+        }
```

Comment:
`XCTAssertEqual` allows a `message` argument: https://developer.apple.com/documentation/xctest/2142776-xctassertequal
```
XCTAssertEqual(url, "wikipedia.org", "Failed to retrieve the URL string from the address toolbar")
```

Reply:
Thank you @clarmso, I wasn't aware of the `message` argument in `XCTAssertEqual`, I've updated the code to include it. Please let me know if there's anything else to adjust